release/readme.md

```diff
@@ -7,6 +7,44 @@
 * run `make tests`, `opam-rt` [checked by github actions]
 * update the CHANGE file: take `master_changes.md` content to fill it
 
+## Windows setup
+
+* Download Windows 10 English International 64bit from https://www.microsoft.com/en-gb/software-download/windows10ISO
+* Run qemu-img create -f qcow2 Windows-10-x86_64.qcow2 32G
+* Run qemu-system-x86_64 -cdrom <path-to-the-windows-iso> -drive file=Windows-10-x86_64.qcow2 -smp 8 -m 6G -nic none
+* Click on Windows 10 Pro
+* Install Windows. Everything stays as default except:
+  * Click "I don’t have a product key"
+  * Click "Custom: ..." not "Upgrade: ..."
+* This will take an hour or so
+* Select the default keyboard then wait some more
+* Upon restart:
+  * no online account
+  * yes, i want the limited experience
+  * username: opam
+  * no password
+  * no location
+  * no find my device
+  * send required diagnostic data
+  * no improve inking & typing
+  * no cortana
+* Open gpedit.msc and right-click on "Computer Configuration -> Administrative Templates -> Windows Components -> Windows Update -> Configure Automatic Updates", click Edit and in the new window click on Enabled and then select "2. Notify for download and auto install", click Apply and then Ok.
+* Start -> Power off
+* Run qemu-system-x86_64 -drive file=Windows-10-x86_64.qcow2 -smp 8 -m 6G
+* Answer yes to the "would you like to be discoverable on the network" prompt
+* Open the Settings app and search for "Add an optional feature", then search for "OpenSSH" and install "OpenSSH Server"
+  * Run services.msc
+  * Double-click OpenSSH SSH Server then select Automatic from the Startup drop-down menu
+  * Click Ok
+* Open cmd.exe as administrator and run "notepad C:/ProgramData/ssh/sshd_config"
+  * Uncomment and set:
+    * PermitEmptyPasswords yes
+    * PermitRootLogin yes
+    * PasswordAuthentication yes
+  * Save and close both applications
+* Open regedit.exe and set HKey_Local_Machine\SYSTEM\CurrentControlSet\Control\Lsa\LimitBlankPasswordUse to 0
```

Comment:
for some reason this step doesn't do what it's supposed to do (allow to ssh in without asking for password on the no-password user) anymore. I need to look into that again.

Reply:
ah StrictMode yes was missing above

Reply:
i'll try again one more time after going through everything but if it's not that it was running "net stop sshd" and "net start sshd", but i highly doubt it